lib/grandpa/message_tracker.go

```diff
@@ -122,3 +127,32 @@ func (t *tracker) handleBlock(b *types.Block) {
 		delete(t.commitMessages, h)
 	}
 }
+
+func (t *tracker) handleTick() {
+	t.mapLock.Lock()
+	defer t.mapLock.Unlock()
+
+	for _, vms := range t.voteMessages {
+		for _, v := range vms {
+			// handleMessage would never error for vote message
+			_, err := t.handler.handleMessage(v.from, v.msg)
+			if err != nil {
+				logger.Debugf("failed to handle vote message %v: %s", v, err)
```

Comment:
honestly, this will never return an error, since `t.handler.handleMessage` for `VoteMessage` never errors. should I just get rid of this check completely?

Reply:
Wellll, for now it never errors, but one could change `handleMessage` and that could blow up in our face :smile: :bomb: 

Maybe just `panic(err)`?

Reply:
I would suggest a `logger.Critical` for now.  After looking at grandpa, it looks like the message is just popped onto the `in` channel and read later via `Service.receiveVoteMessages` where any errors would be logged at debug level.  It would be nice to be able track errors to specific messages that flow through these pipelines.  We could probably track them at the moment with critical log errors.